downgrade

```diff
@@ -159,7 +169,7 @@ search_packages() {
     : "${PACMAN_CACHE:=/var/cache/pacman/pkg/}"
 
     # shellcheck disable=SC2086
-    find $PACMAN_CACHE -maxdepth 1 -regextype posix-extended -regex ".*/$pkgfile_re"
+    find $PACMAN_CACHE -maxdepth $MAXDEPTH -regextype posix-extended -regex ".*/$pkgfile_re"
```

Comment:
`shellcheck disable=SC2086` is here because we _want_ `$PACMAN_CACHE` to word-split if necessary, but `$MAXDEPTH` can (and should) be quoted.

Reply:
I see, will do.